frontend/homework/7/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const cors = require('cors')
+const http = require('http')
+const socketIo = require('socket.io')
+
+const app = express();
+const server = http.createServer(app)
+const io = new socketIo.Server(server, {
+    cors:{
+        origin: "http://127.0.0.1:5500"
+    }
+})
+
+
+app.use(cors());
+app.use(express.json())
+
+io.on("connection", (socket) => {
+    console.log("New user connected")
+    socket.on("message", (payload) => {
+        console.log('Payload', payload)
```

Comment:
??

Reply:
> ??

Sagun, the consoles were just to check if the things are working are not, that is the users are getting connected or not